utils/pod/pod_test.go

```diff
@@ -33,28 +34,24 @@ import (
 func TestLaunchPod(t *testing.T) {
 	// file doesn't exist, should fail
 	files := []string{"docker-fail.yml"}
-	res := LaunchPod(files)
-	if res != types.POD_FAILED {
-		t.Fatalf("expected pod status to be POD_FAILED, but got %s", res)
-	}
+	res, err := LaunchPod(files)
```

Comment:
Does `LaunchPod ` also need to add `ctx`?

Reply:
for LaunchPod, we don't have room to create trace, because all the code is in open source. So the ctx is not required.